code/datums/diseases/advance/symptoms/scalerot.dm

```diff
@@ -0,0 +1,53 @@
+/*
+//////////////////////////////////////
+scalerot
+	Increases resistance.
+	Increases stage speed.
+	Little transmittable.
+	high level
+Bonus
+	Rots away lizards and reverts them to a fleshy form
+//////////////////////////////////////
+*/
+
+/datum/symptom/scalerot
+
+		name = "Scale rot"
+		stealth = 3
+		resistance = 3
+		stage_speed = 3
+		transmittable = 0
+		level = 6
+		severity = 5
+		viable_speciestypes = list(datum/species/lizard)
+
+/datum/symptom/scalerot/Activate(var/datum/disease/advance/A)
+	..()
+			if(prob(SYMPTOM_ACTIVATION_PROB))
+			var/mob/living/M = A.affected_mob
```

Comment:
Bad indentation followed by a (hopefully)unnessary typecast. Viruses should only be in living/carbons anyways.